app/app_utils.cpp

```diff
@@ -16,10 +16,6 @@ bool strtol(const char* nptr, int64_t& n) {
   // strtoll returns LLONG_MAX or LLONG_MIN if an overflow occurs.
   if (tmp == LLONG_MAX || tmp == LLONG_MIN)
     return false;
-  if (tmp < std::numeric_limits<int64_t>::min())
-    return false;
-  if (tmp > std::numeric_limits<int64_t>::max())
-    return false;
```

Comment:
I added this code because `long long` is only required to be _at least_ 64 bits. So it could potentially be 128 bits, in which case these checks are necessary. On platforms where they're unnecessary, the compiler will optimize them away, so there's no harm done.

Reply:
Hrm  OK.